com.kolor.AutopanoPro.yml

```diff
@@ -0,0 +1,52 @@
+app-id: com.kolor.AutopanoPro
+runtime: org.kde.Platform
```

Comment:
do you really need the KDE platform?

Reply:
it is a QT based application, so I thought KDE might be the way to go, 

I can try to use the freedesktop one if that is preferred.

Reply:
you never looked inside the tarball, did you?

It comes with Qt5 prebuilt.

Reply:
Sorry but I did look inside the tarball, but I simply lack the experience to know that it will work just fine on the default freedesktop.Platform which lacks any QT libraries.